bases/recommended.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
```

```suggestion
  "$schema": "https://json.schemastore.org/tsconfig",
```

A cert was recently added so we can use https now 💯